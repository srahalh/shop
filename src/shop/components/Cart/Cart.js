import React from 'react'
import { useSelector } from 'react-redux';

const Cart = () => {
  const { cartCount } = useSelector((state) => state.cart);
  return (
    <div>{cartCount} Articles in ğŸ›’</div>
  )
}

export default Cart