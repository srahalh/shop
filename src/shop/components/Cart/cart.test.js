import React from "react";
import { renderWithProviders } from "../../test/setupTests";
import Cart from "./Cart";
const fulfilledState = require("../../test/fulfilledProductDetailState.json");


describe("Test Cart", () => {
  test("Should render Cart and have count of state in snapshot", () => {
    const { asFragment  } = renderWithProviders(<Cart />, { preloadedState: fulfilledState});
    expect(asFragment ()).toMatchSnapshot();
  });
});
