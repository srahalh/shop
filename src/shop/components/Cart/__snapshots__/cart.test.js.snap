// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Cart Should render Cart and have count of state in snapshot 1`] = `
<DocumentFragment>
  <div>
    2 Articles in ðŸ›’
  </div>
</DocumentFragment>
`;
